### PR TITLE
Add formating instruction for method declaration parameter

### PR DESCRIPTION
**Pull request by benoit-sns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ Intellij IDEA users must install the plugin [Eclipse Code Formatter](http://plug
 
 Go to `Preferences > Editor > General` and check the option `Ensure line feed at file end on Save` (under the `Other` section).
 
+Go to `Preferences > Editor > Code Style > Java > Wrapping and Braces` and uncheck `method declaration parameter > uncheck align when multiline`
```
Comment (julienlancelot) [inline README.md]: Very Minor -> I would rephrase like that : 
Go to ``Preferences > Editor > Code Style > Java > Wrapping and Braces > Method declaration parameter`` and uncheck ``Align when multiline``

Comment (benoit-sns) [reply README.md]: You're right, makes more sense
